testnet/035a2d3600058c877e97006a0e06378fc6ba7b36d89e2f1a39b5a0ec481d264986.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "coinblitz.pro",
```

The validator `id` is missing.

```suggestion
{
  "id": 187,
  "name": "coinblitz.pro",
```